src/components/Layout/Layout.tsx

```diff
@@ -0,0 +1,112 @@
+import React, { ReactNode } from 'react';
+import { Box, Container, makeStyles } from '@material-ui/core';
+
+import { DeveloperProfileGQL } from '../../containers/DeveloperProfile';
+import { useComponentType } from '../../hooks/useComponentType';
+import { DetailsCard } from '../DetailsCard';
+import { Navbar } from '../Navbar';
+import SEO from '../seo';
+
+type LayoutProps = {
+  children: ReactNode;
+  developerProfile: DeveloperProfileGQL;
+  meta: {
+    title: string;
+    description: string;
+    imageUrl: string;
+  };
+  /* eslint-disable react/default-props-match-prop-types, react/require-default-props */
```

Comment:
Disable these rules in the main ESLint file

Reply:
Okay, did that 😄